tests/e2e/api/src/framework/model-repository.ts

```diff
@@ -189,6 +203,21 @@ export interface CreatesModels< T extends ModelRepositoryParams > {
 	create( properties: Partial< ModelClass< T > > ): Promise< ModelClass< T > >;
 }
 
+/**
+ * An interface for repositories that can create child models.
+ *
+ * @typedef CreatesChildModels
+ * @property {CreateChildFn.<T>} create Creates a model using the repository.
```

Comment:
Minor thought (not necessarily worth changing), but we could clarify this is for creating child models:

```suggestion
 * @property {CreateChildFn.<T>} create Creates a child model using the repository.
```

Reply:
Added this.